--- opkssh.debianize/debian/control	2026-06-12 20:02:56.838697341 +0530
+++ opkssh/debian/control	2026-06-12 20:03:19.312189872 +0530
@@ -1,30 +1,83 @@
-Source: golang-github-openpubkey-opkssh
-Section: devel
-Maintainer: Anu-Ra-g <nayakanurag078@gmail.com>
-Build-Depends: debhelper-compat (= 13),
-               dh-sequence-golang,
-               golang-github-docker-go-connections-dev,
-               golang-github-jeremija-gosubmit-dev,
-               golang-github-kballard-go-shellquote-dev,
-               golang-github-melbahja-goph-dev,
-               golang-github-openpubkey-openpubkey-dev,
-               golang-github-spf13-afero-dev,
-               golang-github-spf13-cobra-dev,
-               golang-github-stretchr-testify-dev,
-               golang-go (>= 1.25.0),
-               golang-golang-x-crypto-dev,
-               golang-golang-x-exp-dev,
-               golang-golang-x-mod-dev,
-               golang-golang-x-sys-dev,
-               golang-golang-x-term-dev
-Standards-Version: 4.7.4.1
-Vcs-Browser: https://github.com/openpubkey/opkssh,branch=debian%2Fmain
-Vcs-Git: https://github.com/openpubkey/opkssh,branch=debian%2Fmain.git
-Rules-Requires-Root: no
+Source: opkssh
+Section: golang
+Maintainer: Debian Go Packaging Team <team+pkg-go@tracker.debian.org>
+Uploaders:
+ Simon Josefsson <simon@josefsson.org>,
+Build-Depends:
+ debhelper-compat (= 13),
+ dh-sequence-golang,
+ golang-any,
+ golang-github-docker-go-connections-dev,
+ golang-github-jeremija-gosubmit-dev,
+ golang-github-kballard-go-shellquote-dev,
+ golang-github-lestrrat-go-jwx-dev (>= 3.0.0),
+ golang-github-openpubkey-openpubkey-dev (>= 0.22.0),
+ golang-github-spf13-afero-dev,
+ golang-github-spf13-cobra-dev,
+ golang-github-stretchr-testify-dev,
+ golang-github-thediveo-enumflag-dev,
+ golang-github-zitadel-oidc-dev,
+ golang-golang-x-crypto-dev,
+ golang-golang-x-exp-dev,
+ help2man <!nodoc>,
 Testsuite: autopkgtest-pkg-go
+Standards-Version: 4.7.4
+Vcs-Browser: https://salsa.debian.org/go-team/packages/opkssh
+Vcs-Git: https://salsa.debian.org/go-team/packages/opkssh.git
+Homepage: https://github.com/openpubkey/opkssh
 XS-Go-Import-Path: github.com/openpubkey/opkssh
 
+Package: opkssh
+Section: net
+Architecture: any
+Depends:
+ ${misc:Depends},
+ ${shlibs:Depends},
+Built-Using:
+ ${misc:Built-Using},
+Static-Built-Using:
+ ${misc:Static-Built-Using},
+Description: OpenPubkey SSH tools for OpenID SSH logins (program)
+ opkssh (OpenPubkey SSH) is a tool which enables ssh to be used with
+ OpenID Connect allowing SSH access management via identities like
+ alice@example.com instead of long-lived SSH keys. It does not replace
+ ssh, but rather generates ssh public keys that contain PK Tokens and
+ configures sshd to verify the PK Token in the ssh public key. These
+ PK Tokens contain standard OpenID Connect ID Tokens. This protocol
+ builds on the OpenPubkey
+ (https://github.com/openpubkey/openpubkey/blob/main/README.md) which
+ adds user public keys to OpenID Connect without breaking
+ compatibility with existing OpenID Provider.
+ .
+ This package contains the command-line tool 'opkssh'.
+
 Package: golang-github-openpubkey-opkssh-dev
 Architecture: all
 Multi-Arch: foreign
-Depends: ${misc:Depends}
+Depends:
+ golang-github-docker-go-connections-dev,
+ golang-github-jeremija-gosubmit-dev,
+ golang-github-kballard-go-shellquote-dev,
+ golang-github-lestrrat-go-jwx-dev (>= 3.0.0),
+ golang-github-openpubkey-openpubkey-dev (>= 0.22.0),
+ golang-github-spf13-afero-dev,
+ golang-github-spf13-cobra-dev,
+ golang-github-stretchr-testify-dev,
+ golang-github-thediveo-enumflag-dev,
+ golang-github-zitadel-oidc-dev,
+ golang-golang-x-crypto-dev,
+ golang-golang-x-exp-dev,
+ ${misc:Depends},
+Description: OpenPubkey SSH tools for OpenID SSH logins (library)
+ opkssh (OpenPubkey SSH) is a tool which enables ssh to be used with
+ OpenID Connect allowing SSH access management via identities like
+ alice@example.com instead of long-lived SSH keys. It does not replace
+ ssh, but rather generates ssh public keys that contain PK Tokens and
+ configures sshd to verify the PK Token in the ssh public key. These
+ PK Tokens contain standard OpenID Connect ID Tokens. This protocol
+ builds on the OpenPubkey
+ (https://github.com/openpubkey/openpubkey/blob/main/README.md) which
+ adds user public keys to OpenID Connect without breaking
+ compatibility with existing OpenID Provider.
+ .
+ This package contains the Go development library.
