--- python-aiohttp.debianize/debian/upstream/metadata	2026-06-12 19:51:29.074902846 +0530
+++ python-aiohttp/debian/upstream/metadata	2026-06-12 19:51:37.470259520 +0530
@@ -1,4 +1,5 @@
----
-Name: aiohttp
-Contact: aiohttp team <team@aiohttp.org>
 Bug-Database: https://github.com/aio-libs/aiohttp/issues
+Bug-Submit: https://github.com/aio-libs/aiohttp/issues/new
+Changelog: https://github.com/aio-libs/aiohttp/blob/master/CHANGES.rst
+Repository: https://github.com/aio-libs/aiohttp.git
+Repository-Browse: https://github.com/aio-libs/aiohttp
